app/controllers/concerns/importable.rb

```diff
@@ -27,8 +27,12 @@ def import_csv
       data = File.read(params[:file].path, encoding: "BOM|UTF-8")
       csv = CSV.parse(data, headers: true).reject { |row| row.to_hash.values.any?(&:nil?) }
       if csv.count.positive? && csv.first.headers.all? { |header| !header.nil? }
-        resource_model.import_csv(csv, current_organization.id)
-        flash[:notice] = "#{resource_model_humanized} were imported successfully!"
+        errors = resource_model.import_csv(csv, current_organization.id)
+        if errors.nil?
```

Comment:
I'd prefer `errors` returning an empty array and checking `errors.empty?`. There's no semantic difference between an empty array and nil, and this way we don't have to have a special "return nil" case further down.